app/backend/authentication/views.py

```diff
@@ -38,13 +38,44 @@ class UserViewSet(viewsets.ModelViewSet):
 @permission_classes([AllowAny])
 def register_user(request):
     try:
+        custom_data = {
```

Comment:
We may want to communicate with other teams about the exception handling to avoid further problems.

Reply:
Good point!